### PR TITLE
unit test

### PR DESCRIPTION
**Pull request by huyencao**
https://edu-redmine.sun-asterisk.vn/issues/13956

### CODE REVIEW

**Comment by huyencao** [comment]
ready

**Comment by huyencao** [comment]
file:///home/caohuyen/Pictures/Screenshot%20from%202019-07-01%2008-56-04.png

**Comment by huyencao** [comment]
Screenshot from 2019-07-01 08-56-04

File: tests/Unit/UserTest.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+     */
+    public function testCreate()
+    {
+        $faker = Factory::create();
+        $data = [
+            'username' => $faker->username,
+            'email' => $faker->unique()->safeEmail,
+            'avatar_id' => rand(1, 10),
+            'password' => bcrypt('realestate'),
+            'role' => rand(1,3),
```
Comment (sonnguyen8x) [inline tests/Unit/UserTest.php]: convention

File: tests/Unit/UserTest.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    public function testDetele()
+    {
+        $faker = Factory::create();
+        $data = [
+            'username' => $faker->username,
+            'email' => $faker->unique()->safeEmail,
+            'avatar_id' => rand(1, 10),
+            'password' => bcrypt('realestate'),
+            'role' => rand(1,3),
```
Comment (sonnguyen8x) [inline tests/Unit/UserTest.php]: convention

**Comment by huyencao** [comment]
ready

File: config/database.php [hunk 3]
```diff
@@ -84,6 +84,18 @@
             'prefix' => '',
             'prefix_indexes' => true,
         ],
+        //cấu hình kết nối data test cho
+        'mysql_test' => [
+            'driver'    => 'mysql',
```
Comment (BunnyPi04) [inline config/database.php]: xóa space dư nhé

**Comment by huyencao** [comment]
ready

**Pull request merged**